api/v1beta1/openstackcluster_types.go

```diff
@@ -18,7 +18,7 @@ package v1beta1
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	clusterv1 "sigs.k8s.io/cluster-api/api/core/v1beta2"
```

Comment:
I would suggest keep using v1beta1 here and also in other files that are in API folder: 
```suggestion
	clusterv1beta1 "sigs.k8s.io/cluster-api/api/core/v1beta1"
```